src/zenml/integrations/seldon/model_deployers/seldon_model_deployer.py

```diff
@@ -470,3 +476,46 @@ def delete_model_server(
         # secret used to store the authentication information for the Seldon
         # Core model server storage initializer
         self._delete_kubernetes_secret()
+
+    def prepare_custom_deployment_image(
```

Comment:
duplicate code with kserve?

Reply:
Yes, I had a discussion with @schustmi  about this. since not all deployers have to do this step and it will be different definition for bentoML and MLFlow it is okay to have it duplicated